corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -564,15 +564,30 @@ def _migrate_form_attachments(sql_form, couch_form):
     attachments = []
     metadb = get_blob_db().metadb
     for name, blob in six.iteritems(couch_form.blobs):
+        def try_to_get_blob_meta(parent_id, type_code, name):
```

Comment:
nit: what do you think of moving this function definition out of the for-loop?

Reply:
e0c8763